### PR TITLE
Refactor redirect paths for Kanvas and Meshmap

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 281bc634e6b50d5c1a6686528a4895c9e8a763ea at: https://66dfb7d77c45f2ec2c2fe4e4--layer5.netlify.app

File: gatsby-node.js [hunk 1]
```diff
@@ -284,6 +284,18 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+    toPath: "/cloud-native-management/meshmap",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+  createRedirect({
+    fromPath: "/cloud-native-management/meshmap",
+    toPath: "/cloud-native-management/kanvas",
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
```
Comment (vishalvivekm) [inline gatsby-node.js:298]:
```suggestion
  createRedirect({
    fromPath: "/kanvas",
    toPath: "/cloud-native-management/meshmap",
    redirectInBrowser: true,
    isPermanent: true,
  });
  createRedirect({
    fromPath: "/cloud-native-management/kanvas",
    toPath: "/cloud-native-management/meshmap",
    redirectInBrowser: true,
    isPermanent: true,
  });
```

Comment (vishalvivekm) [reply gatsby-node.js:298]: we are looping from 
`/kanvas` to `/cloud-native-management/meshmap` to `/cloud-native-management/meshmap` to `/cloud-native-management/kanvas`

**Comment by l5io** [comment]
🚀 Preview for commit 4f20383bdf474fd80bb11550c3450bc165a4419e at: https://66dfbeb08e87f4ee78722b7b--layer5.netlify.app

**Comment by leecalcote** [comment]
Other changes made and pushed.

**Comment by l5io** [comment]
🚀 Preview for commit 61b0cf4a0bd0877626c62e142770b50836967bb6 at: https://66e04f90fd50be3a5088ebe6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f58bff5fb6a7893418655f0d1f881a9e10c53a5e at: https://66e07cd87e444a7603e6ffb3--layer5.netlify.app

**Pull request merged**